site/gatsby-site/src/components/variants/VariantsTable.js

```diff
@@ -0,0 +1,440 @@
+import { useUserContext } from 'contexts/userContext';
+import React, { useState } from 'react';
+import Markdown from 'react-markdown';
+import { Button, Form, Pagination } from 'react-bootstrap';
+import { format, getUnixTime } from 'date-fns';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { useMutation } from '@apollo/client';
+import { useBlockLayout, useFilters, usePagination, useResizeColumns, useTable } from 'react-table';
+import VariantEditModal from './VariantEditModal';
+import styled from 'styled-components';
+import useToastContext, { SEVERITY } from '../../hooks/useToast';
+import { Trans, useTranslation } from 'react-i18next';
+import { VariantStatusBadge } from './VariantList';
+import { VARIANT_STATUS } from 'utils/variants';
+import { DELETE_VARIANT, UPDATE_VARIANT } from '../../graphql/variants';
+import { LINK_REPORTS_TO_INCIDENTS } from '../../graphql/reports';
+
+const Table = styled.div`
```

Comment:
We're trying to reduce the use of `styled-components`, is there a way you can replace this with regular html (like on the entities page)? Otherwise in the future we'll need to replace it.

Reply:
Done!